src/django/api/serializers/v1/moderation_event_update_serializer.py

```diff
@@ -13,8 +13,8 @@ class ModerationEventUpdateSerializer(ModelSerializer):
 
     contributor_id = IntegerField(source='contributor.id', read_only=True)
     contributor_name = CharField(source='contributor.name', read_only=True)
-    os_id = IntegerField(source='os.id', read_only=True, allow_null=True)
-    claim_id = IntegerField(source='claim.id', read_only=True, allow_null=True)
+    os_id = IntegerField(source='os.id', read_only=True)
+    claim_id = IntegerField(source='claim.id', read_only=True)
```

Comment:
Could you please also disable the possibility of a null value for the source field, as it could potentially be null in the case of "future" claim moderation events?

Reply:
Done, pls check.